PassTime.php

```diff
@@ -0,0 +1,19 @@
+<?php
+import "header.php";
+import "classes.php";
+import "commonUI.php";
+
+
+$time = $_GET["$restDays"];
+
+for ($i=0; $i < $time; $i++) { 
+    $_SESSION["journey"]->incrementDay();
+}
+
+$dateString = $_SESSION["journey"]->$_month . " " . $_SESSION["journey"]->$_date;
```

Comment:
The agreed-upon key for the journey is "playerJourney". Also if you do arrow-syntax to a member variable you don't need the dollar sign in front of the member name.

You can see the agreed-upon keys in `$_SESSION` in the pinned post on slack under #general.